cmd/podman/main_local.go

```diff
@@ -113,6 +114,35 @@ func setupRootless(cmd *cobra.Command, args []string) error {
 		MainGlobalOpts,
 		remoteclient,
 	}
+
+	pausePidPath, err := util.GetRootlessPauseProcessPidPath()
```

Comment:
Can we start moving things like this block of code out of `cmd/podman` and consolidating them someplace? Maybe `pkg/rootless`? I want to make it easier to integrate this into other Libpod consumers

Reply:
yes, I agree.  The only reason I've left it here for now, is that the logic for detecting running containers and try to join their namespaces (and this is particularly important to not break rootless containers once again on an upgrade) cannot be easily moved around.

Reply:
I had a look at it now, but there are parts that cannot be easily moved down to the pkg/rootless package.  For example we depend on `util.GetRootlessRuntimeDir()` and in the case of not having a pause process running, we need to retrieve a list of conmon processes.